### PR TITLE
TG 60: Admins

### PR DESCRIPTION
**Pull request by OctavioRodriguezPaseyro**
This pull request will be merged when all the shared components are correctly added to it.

### CODE REVIEW

File: src/Components/Admins/Form/form.module.css [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+}
+
 .container {
   flex: 1;
   display: flex;
-  flex-direction: column;
+  text-align: center;
+  margin: 1%;
+}
+
```
Comment (gavafue) [inline src/Components/Admins/Form/form.module.css:15]: Remember keep the consistance of the CSS, .container should to be before to .adminForm

File: src/Components/Admins/Form/index.js [hunk 2]
```diff
@@ -97,84 +94,79 @@ function Form() {
...
       const data = await res.json();
-      if (res.status == 201 || res.status == 200) {
-        setContentFeedbackModal({ title: 'Request done!', description: data.message });
-        setShowFeedbackModal(true);
+      if (data.status == 201 || data.status == 200) {
+        alert(data.message);
       } else {
-        setContentFeedbackModal({ title: 'Something went wrong', description: data.message });
-        setShowFeedbackModal(true);
+        alert(data.message);
```
Comment (gavafue) [inline src/Components/Admins/Form/index.js:100]: What is the function of this alerts?

File: src/Components/Projects/Table/index.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ const ProjectsTable = ({
                 <td>{project.client}</td>
                 <td>{project.startDate}</td>
                 <td>{project.endDate}</td>
-                <td>{project.members[0]._id}</td>
+                <td>{project.members.name}</td>
```
Comment (gavafue) [inline src/Components/Projects/Table/index.js:46]: Thank you for solve the projects' entity problem

**Review by gavafue — COMMENTED** [review]
Check it that you have some comments ando conflicts on your PR

**Review by gavafue — APPROVED** [review]

**Comment by paurinaldi** [comment]
I see that Gabi already approved but there are two things I noticed: Your form is a row, it should be a column, as the ones on figma we said we were going to base our styling on. Also, the add button is in a totally different color and format as all the others, we agreed add and submit would use the Button's scondary style.

File: src/Components/Admins/Form/form.module.css [hunk 4]
```diff
@@ -1,7 +1,27 @@
...
   flex-direction: column;
   justify-content: center;
   align-items: center;
-}
\ No newline at end of file
+  text-align: center;
+  margin: 1%;
+  border:1px solid #999;
+	border-radius:25px;
+	box-shadow:0 0 10px #999;
```
Comment (samueltrilloRR) [inline src/Components/Admins/Form/form.module.css:26]: Bad indentation

File: src/Components/Admins/Form/index.js [hunk 5]
```diff
@@ -1,6 +1,12 @@
...
-import FeedbackModal from '../FeedbackModal';
+import { useParams } from 'react-router-dom';
+import Button from '../../Shared/Button';
+import Input from '../../Shared/Input/InputText';
+import Select from '../../Shared/Input/InputSelect';
+import Modal from '../../Shared/Modal';
+import FeedbackMessage from '../../Shared/FeedbackMessage';
+import Preloader from '../../Shared/Preloader';
 
 function Form() {
```
Comment (samueltrilloRR) [inline src/Components/Admins/Form/index.js:11]: Transform function into arrow function (EM6)

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

File: src/Components/Admins/index.js [hunk 6]
```diff
@@ -1,47 +1,31 @@
...
-        deleteAdmin={deleteAdmin}
-        showModal={showModal}
-        setShowModal={setShowModal}
-      />
-      <button /* className={styles.addBtn} */>
-        <a href="/admins/form">Add New Admin</a>
-      </button>
+    <section className={styles.container}>
+      <h2>Admins</h2>
+      <Button label="Add new admin +" onClick={() => (window.location = `/admins/form/`)} />
```
Comment (samueltrilloRR) [inline src/Components/Admins/index.js]: window.location should be replaced with React router hook

**Review by samueltrilloRR — CHANGES_REQUESTED** [review]

File: src/Components/Admins/index.js (hunk 6 above)
Comment (OctavioRodriguezPaseyro) [reply src/Components/Admins/index.js]: line 26 was changed on last commit "TG-60: correction"

<Button label="Add new admin +" onClick={() => history.push(`/admins/form/`)} />

**Review by samueltrilloRR — APPROVED** [review]

**Pull request merged**